bpfd/src/bpf.rs

```diff
@@ -553,4 +561,163 @@ impl<'a> BpfManager<'a> {
         }
         results
     }
+
+    pub(crate) async fn process_commands(&mut self) {
+        loop {
+            // Start receiving messages
+            select! {
+                biased;
```

Comment:
What is this doing?

Reply:
Ensures that the shutdown branch gets priority.